### DISCUSSION TITLE
How to solve CORS error on redirect

### DISCUSSION THREAD

**Discussion by StokleyO** [sveltejs/kit#10982] [Q&A]
Hey everyone!

On my SvelteKit app, I use a Stripe Connect integration that allows users to transact on my site. I had a throw redirect that was working previously but suddenly has recently stopped working. From what I can tell when viewing the network tab, it looks like there's a CORS error. I've tried creating a hooks.server.ts file where I try to get around the CORS restrictions using [this as a guide](https://dev.to/khromov/configure-cors-in-sveltekit-to-make-fetch-requests-to-your-api-routes-from-a-different-host-241k). Does anyone have any ideas on how to get around CORS issues in SvelteKit? OR why this redirect was working previously and now suddenly does not work?

Any help whatsoever would be greatly appreciated

**Comment by StokleyO** [comment]
Something to note as well, when I switch out the redirect path to another page on my site, in the network tab I get a 200, but it never redirects

**Reply by Scott3730** [reply]
I'm experiencing this exact issue now using Stripe and SvelteKit, trying to find a solution.
